app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -57,13 +56,6 @@ def pending_profile_policy
       )
     end
 
-    def block_biometric_requests_in_production
-      if biometric_comparison_requested? &&
-         !FeatureManagement.idv_allow_selfie_check?
-        render_not_acceptable
-      end
-    end
```

Comment:
@jmhooper So, I ripped this out which I think was right, but I'm unsure how to deal with tests that reduce down to this:

```ruby
  context 'selfie capture not enabled, biometric_comparison_check requested by sp' do
    let(:selfie_capture_enabled) { false }
    it 'returns status not_acceptable' do
      action
  
      expect(response.status).to eq(406)
    end
  end
```

They no longer pass, but `selfie_capture_enabled` is passed to `doc_auth_selfie_capture`, which has now been simplified to be set to `resolved_authn_context_result.biometric_comparison?`.

I'm thinking that this really means that the meaning of `let(:selfie_capture_enabled) { false }` has changed (formerly it was intending to set to false the feature flag). Am I good to just rip these tests out?

Reply:
The behavior here is intended to render a 406 if an SP requests biometric comparison and the environment is not configured to allow it. That is no longer relevant once this flag is removed. Even if we do implement something like a per-SP allow list for biometric comparison I suspect we'll want different behavior so this should be safe to remove.